lib/java/com/google/android/material/lists/viewHolders/threeLine/ThreeLineOverlineItemViewHolder.java

```diff
@@ -0,0 +1,45 @@
+package com.google.android.material.lists.viewHolders.threeLine;
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.material.R;
+import com.google.android.material.lists.viewHolders.MaterialViewHolder;
+
+
+/**
+ * A three-lined ViewHolder with a overline text, title text, and two lines of subtext
```

Comment:
Replace the word "a" before "overline text" with "an":

```suggestion
 * A three-lined ViewHolder with an overline text, title text, and two lines of subtext
```